protocol/0056-REWA-rewards_overview.md

```diff
@@ -215,6 +215,28 @@ This flag is used to prevent any given funder from funding a creation reward in
 
 Note this reward metric **is not** available for team rewards.
 
+## Eligible entities reward metric
+
+The eligible entities rewards metric $m_ee$ rewards entities who meet all the [eligibility requirements](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts) set in the transfer.
+
+If a party meets **all** the eligibility requirements, their reward metric $m_ee$ is simply set to $1$.
+
+### Conditional transfer fields
+
+In order to allow creation of rewards which can pay-out to parties who are not actively trading, a transfer using this metric should be accepted in the cases where:
+
+- it specifies no metric asset, no markets within the market scope, no staking requirement, and no position requirement - in this case, all parties on the network are given a score of $1$.
+- it specifies no metric asset, no markets within the market scope, no position requirement, but does specify a staking requirement - in this case, all parties meeting the staking requirement are given a score of $1$.
+
+If however a position requirement is specified, an asset must be specified also and then parties must meet the position requirement to receive rewards.
+
+### Reward windows and transfer delays
+
+Note the following interactions with reward windows and transfer delays.
+
+- if a reward window greater than one is specified, an entity only needs to meet the eligibility requirements in the epoch the reward is distributed.
+- if a transfer delay greater than one is specified, an entity still **only** needs to meet the eligibility requirements in the epochs in the reward window (as per the point above).
```

Comment:
If the change to check all epochs in the reward window is made, this sentence reads correctly. However as currently specified they only need to meet the requirements in the distribution epoch, not the epochs in the reward window.

Reply:
Should be addressed now.